python/shipDet_conf.py

```diff
@@ -329,6 +329,9 @@ def configure(run, ship_geo):
         SC_mag=ship_geo.SC_mag,
         shieldName=ship_geo.shieldName,
         DecayVolumeMedium=ship_geo.DecayVolumeMedium,
+        SND=ship_geo.SND,
+        SND_design=ship_geo.SND_design,
+        TARGET_YAML=ship_geo.target_yaml
```

Comment:
This is unrelated to the helium. We should address this separately, ideally by removing this reloading of the geometry_config altogether.

Reply:
So I think that right now it is needed to take the parameters from the run_simScript, otherwise they will be ignored here !! But I agree it is a little clumsy. It is like it was used to add parameters to this file only without the need to add them to the run_simScript...